x-pack/plugin/esql/qa/testFixtures/src/main/resources/mapping-multivalues.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I realized that this requires larger changes since `employees.csv` does not seem to have doubles with duplicate multivalues. I'll go with a yaml test instead.